### ISSUE TITLE
Watch 3Blue1Brown in class?

### ISSUE THREAD

**Issue by joeyklee** [shiffman/Intro-ML-Arts-IMA#27]
https://www.youtube.com/watch?v=aircAruvnKk&feature=youtu.be 

I've found this video to be understandable to even people who haven't worked with ML before and helps demystify what is a "neural network". Maybe people should watch before coming to class next time to help make some of these concepts like gradient descent, hidden layers, etc etc? 

While the video is nearly 20min could be helpful to watch together so that students can ask questions along the way? or maybe helpful if students have to respond to the video with an illustration of how all those components work?

**Comment by shiffman** [comment]
A helpful reminder that students should watch this video between Monday and Wednesday this week. I agree that stopping and starting the video with questions / discussion would make it much more meaningful to watch in class, maybe I can pick out some parts of we can watch it on Wed this week.
